src/intarface/interface.ts

```diff
@@ -0,0 +1,62 @@
+export interface ICharacter {
+    comics: IComics;
+    description: string;
+    events: IEvents;
+    id: number;
+    modified: string;
+    name: string;
+    resourceURL: string;
+    series: ISeries;
+    stories: IStories;
+    thumbnail: IThumbnail;
+    urls: IUrls[];
+}
+
+export interface IComics {
```

Comment:
so many repeatable properties in interfaces and some interfaces are exactly the same, rewrite them